templates/zerver/register.html

```diff
@@ -196,6 +196,24 @@ <h1>{{ _('Create your account') }}</h1>
             <hr />
             {% endif %}
 
+            {% if creating_new_realm %}
+            <div class="input-group input-box" id="how-realm-creator-found-zulip">
+                <label for="how_realm_creator_found_zulip">
+                    {{ _('How did you first hear about Zulip?') }}
+                    <i class="fa fa-question-circle-o" aria-hidden="true" data-tippy-content="{% trans %}This value is unused unless you sign up for a plan, in which case it will be sent to the Zulip team.{% endtrans %}"></i>
```

Comment:
Let's get rid of the double negative, since that's a bit hard to parse.

> This value is used only if you sign up for a plan, in which case it will be sent to the Zulip team.